<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>后台封边程序转化</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/tup.png</normaloff>images/tup.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>统计</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="MprProcessing">
          <property name="text">
           <string>Mpr处理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Buttonbanding2">
          <property name="text">
           <string>五楼封边</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Buttonbanding">
          <property name="text">
           <string>四楼封边</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="PatchActionc"/>
    <addaction name="sqlActionc"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="VersionAction"/>
    <addaction name="ExitAction"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="PatchActionc"/>
   <addaction name="ExitAction"/>
   <addaction name="sqlActionc"/>
  </widget>
  <action name="PatchActionc">
   <property name="icon">
    <iconset>
     <normaloff>images/loadtest.png</normaloff>images/loadtest.png</iconset>
   </property>
   <property name="text">
    <string>路径设置(&amp;T)</string>
   </property>
   <property name="statusTip">
    <string>设置相关文件路径</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="VersionAction">
   <property name="text">
    <string>版本号(&amp;U)</string>
   </property>
   <property name="toolTip">
    <string>版本号(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="ExitAction">
   <property name="icon">
    <iconset>
     <normaloff>images/exit.png</normaloff>images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="sqlActionc">
   <property name="icon">
    <iconset>
     <normaloff>images/456.jpg</normaloff>images/456.jpg</iconset>
   </property>
   <property name="text">
    <string>SqlServer设置（&amp;Y）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
